lib/ansible/modules/cloud/digital_ocean/digital_ocean_certificate.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_certificate
+short_description: Manage certificate in DigitalOcean.
+description:
+    - Create, Retrieve and remove certificates DigitalOcean.
+author: "Abhijeet Kasurde (@akasurde)"
+version_added: "2.4"
+options:
+  name:
+    description:
+     - The name of the certificate.
+    required: true
+  private_key:
+    description:
+    - A PEM-formatted private key content of SSL Certificate.
+  leaf_certificate:
+    description:
+    - A PEM-formatted public SSL Certificate.
+  certificate_chain:
+    description:
+    - The full PEM-formatted trust chain between the certificate authority's certificate and your domain's SSL certificate.
+  state:
+    description:
+     - Whether the certificate should be present or absent.
+     - If set to C(list_all), will list all SSL certificates.
```

Comment:
It will highlight the enclosed string in documentation - just like `ami` [here](http://docs.ansible.com/ansible/latest/ec2_ami_search_module.html#synopsis)